### ISSUE TITLE
Some remarks on using the `plot_topoplotseries` function

### ISSUE THREAD

**Issue by jschepers** [unfoldtoolbox/UnfoldMakie.jl#153]
I recently used the `plot_topoplotseries` function and have some remarks:

1. I was not sure what kind of vector I was supposed to be using for the `positions` argument. In the documentation, it only says `Shows channel positions.`. I was unsure between a position for each data point or a list of all (unique) positions (the latter is the case but I didn't know at the beginning). There it might be helpful to adapt the docstring.
2. I struggled a bit with finding the right parameters to change to have a decent-looking figure and which values to choose for them. In the end, I modified `enlarge`, `label_scatter`, figure size and manually changed the x- and y axis limits. Before this. the plot looked like this: 
![grafik](https://github.com/unfoldtoolbox/UnfoldMakie.jl/assets/22366977/deae7a34-e9b3-4fd7-8bee-c03f8e85d1d7)
TL;DR: It would be nice to have some guidance on setting these parameters.
3. As said in the former bullet point, I adapted the x-and y-axis limits. I did it in the following way:
```julia
ax = current_axis()
linkaxes!(tp_effects.content[1:end-2]...)
xlims!(ax, 0, 0.9)
ylims!(ax, 0, 0.9)
current_figure()
```
It would be nice if this would be possible as part of the `plot_topoplotseries` function e.g. in `axis`.
4. It would be nice to have labels for the overall x- and y-axis e.g. "time windows [s]" for the x-axis and "variablename" on the y-axis. Bene suggested to have it in `axis` i.e. `axis = (; xlabel = "...",  ylabel = "..." )`.

**Comment by vladdez** [comment]
Hi! @jschepers can you please send your final code?

**Comment by jschepers** [comment]
Hi, what do you mean by "final code", @vladdez ? 
Do you mean the whole code to create this plot?

**Comment by vladdez** [comment]
> Hi, what do you mean by "final code", @vladdez ? Do you mean the whole code to create this plot?

yep

**Comment by jschepers** [comment]
It's rather long because it involves simulating multi-subject data, fitting Unfold models for all subjects, computing marginal effects, aggregating them over subjects and then plotting them in a topoplot series. I am finalising the tutorial that I'm using the code for and will post the link when I've finished it but my points should apply also to other examples e.g. from the UnfoldMakie docs.

**Comment by jschepers** [comment]
https://github.com/unfoldtoolbox/UnfoldStats.jl/blob/first-tutorial/docs/literate/tutorial/two_stage_analysis.jl

At the moment, it's still at the branch `first-tutorial` not on the main branch.

**Comment by vladdez** [comment]
Should we even allow users to specify `colorrange`? Or it should be always data dependant?

**Comment by vladdez** [comment]
> colorrange should definitely be adjustable, many good reasons why it might fail
> 
> Edit: what is the relevance of colorrange for this issue though?

because Judith specified it
